guides/source/debugging_rails_applications.md

```diff
@@ -710,7 +710,7 @@ In addition to different types of breakpoints, you can also specify options to a
 - `if: <expr>` - the breakpoint only stops if the result of `<expr`> is true:
   - `break Post#save if: params[:debug]` - stops at `Post#save` if `params[:debug]` is also true
 - `path: <path_regexp>` - the breakpoint only stops if the event that triggers it (e.g. a method call) happens from the given path:
-  - `break Post#save if: app/services/a_service` - stops at `Post#save` if the method call happens at a method matches Ruby regexp `/app\/services\/a_service/`.
+  - `break Post#save path: app/services/a_service` - stops at `Post#save` if the method call happens at a method matches Ruby regexp `/app\/services\/a_service/`.
```

Comment:
We probably would also want to update the Post#save usage  or the regex path as they don't seem correct too

Reply:
Yes, I see what you mean. Amended my commit to propose some new wording.

Here's the [debug docs](https://github.com/ruby/debug?tab=readme-ov-file#breakpoint), for reference.